web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/page/BasicInformation/index.tsx

```diff
@@ -157,7 +159,9 @@ export const BasicInformation = (props: {
             type="submit"
             variant="primary"
             className=" mr-5 w-40 h-12"
-            disabled={isSubmitting}
```

Comment:
@andy-t-wang Why do we exclude `isSubmitting`?

Reply:
I thought the button looked dumb turning disabled so I put the check inside of the submit callback